android/WALT/app/src/main/java/org/chromium/latency/walt/ClockManager.java

```diff
@@ -97,12 +99,29 @@ private ClockManager(Context context) {
         mContext = context;
         mUsbManager = (UsbManager) mContext.getSystemService(Context.USB_SERVICE);
         mLogger = SimpleLogger.getInstance(context);
+        mBroadcastManager = LocalBroadcastManager.getInstance(context);
     }
 
     public boolean isConnected() {
         return ((mEndpointIn != null) && (mEndpointOut != null));
     }
 
+    public void registerConnectCallback(final Runnable r) {
+        if (isConnected()) {
+            r.run();
+            return;
+        }
+
+        mLogger.log("Registering receiver");
```

Comment:
Please add something about what receiver this is talking about. I think by now we have several things called "receiver" in the system.